app/models/miq_provision_workflow.rb

```diff
@@ -3,17 +3,17 @@ def self.base_model
     MiqProvisionWorkflow
   end
 
+  # Find the appropriate workflow class for the given +platform+ string. For
```

Comment:
Should this be in yard doc format? https://github.com/ManageIQ/guides/blob/d5173b29cde420ede2881a1721c7c2aff4083aa6/coding_style_and_standards.md#documentation

Reply:
Also, this method seems simple enough to not require documentation at this point.

Reply:
@bdunne ok, updated to use yard. I really should force myself to start using that more anyway.